pkg/actors/actors.go

```diff
@@ -1152,6 +1168,40 @@ func (a *actorsRuntime) doExecuteReminderOrTimer(ctx context.Context, reminder *
 }
 
 func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderRequest) error {
+	if a.scheduler != nil {
+		metadata := map[string]string{
+			"appId":        a.actorsConfig.AppID,
+			"actorType":    req.ActorType,
+			"actorId":      req.ActorID,
+			"reminder":     req.Name,
+			"content-type": "application/json",
+		}
+
+		jobName := constructCompositeKey(
+			"reminder",
+			req.ActorType,
+			req.ActorID,
+			req.Name,
+		)
+
+		// TODO: change the 3rd party library to take our format
+		jobSchedule := "@every " + req.Period
+		internalScheduleJobReq := &schedulerv1pb.ScheduleJobRequest{
+			Job: &runtimev1pb.Job{
+				Name:     jobName,
```

Comment:
Can you still prepend the AppID to the start of the job name? This will be needed for the parsing of jobs per appID later for the `ListJobs()`. I can switch the list to filter the appID with the `||` instead of `_`, but the appID will need to be the first thing in the jobname

```suggestion
				Name:     fmt.Sprintf("%s||%s", a.actorsConfig.AppID, jobName),
```